src/csharp/Grpc.Core/SourceLink.csproj.include

```diff
@@ -0,0 +1,19 @@
+<!-- Ensure that debugging of the resulting NuGet packages work (we're using SourceLink). -->
+<Project>
+
+  <ItemGroup Label="dotnet pack instructions">
+    <Content Include="$(OutputPath)netstandard1.5\$(PackageId).pdb">
+      <Pack>true</Pack>
+      <PackagePath>lib/netstandard1.5</PackagePath>
```

Comment:
nit: just a guess, is the use of `/` necessary in the path here? This is inconsistent with the backslash separators used elsewhere?